app/models/user.rb

```diff
@@ -34,7 +35,13 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable
   include DeviseTokenAuth::Concerns::User
 
+  MAX_ALLOWED_TARGETS = 10
+
   enum gender: { female: 0, male: 1, fluid: 2 }
   has_many :targets, dependent: :destroy
   validates :gender, presence: true
+
+  def can_add_more_targets?
+    targets.count >= MAX_ALLOWED_TARGETS
+  end
```

Comment:
I probably move this to the Target model, and call the constant like `User::MAX_ALLOWED_TARGETS`, so we could keep it private